code/game/objects/structures/transit_tubes.dm

```diff
@@ -672,4 +703,18 @@
 		else
 	return
 
+/obj/structure/transit_tube/proc/iconstate2framedir(var/icon_state)
```

Comment:
looks like the `icon_state` parameter is always equal to `src.icon_state`. Change all callers as well.
```suggestion
/obj/structure/transit_tube/proc/iconstate2framedir()
```

Reply:
Fixed